### ISSUE TITLE
Mouseover tip labels sometimes cut off from view

### ISSUE THREAD

**Issue by emmahodcroft** [nextstrain/auspice#1356]
**Current Behavior**  
Depending on tip position and tree view, sometimes mouseover tip labels are cutoff so you can't see the entire box. Since these mouseover boxes contain some info that's not in the tip-click boxes (like divergence), it makes this information not-accessible. Also if trying to see a bunch of tips, can be time-consuming to click them all. 

Understandable that this won't be possible at all zooms and screen resolutions, but below is at 100% browser zoom in 1920x1200 resolution, full screen (laptop), and it seems like if box was moved up a bit, at least a lot more of it could be visible:
![image](https://user-images.githubusercontent.com/14290674/121501612-aea6f780-c9df-11eb-9c93-a4e261c98518.png)
https://nextstrain.org/ncov/europe?c=S1_mutations&l=clock&s=India/MH-NIV-8404/2020,Mongolia/7908/2020,Peru/LIM-INS-768/2020,France/PAC-IHU-3126i/2020

**Expected behavior**  
Maybe reduce the amount trying to link tool-tip mouseover-box to 'start' at the node being moused over, and let it be at a higher point if more of the box would be visible?

However, I know tool-tip boxes can be a real pain to get right. Still thought it was worth flagging as I keep running into it!! Ensuring all info in mouse-over boxes is also in tip-click boxes would also help.

**Your environment: if browsing Nextstrain online**  
 - Operating system:  Windows 10 Pro
 - Browser:  Chrome

**Comment by trvrb** [comment]
I agree that this is annoying. Two thoughts:

1. We should definitely make sure that everything listed in `infoPanel.hover` is available in `infoPanel.click`. Easy to add "divergence" here. Were there other attributes missing? I think it's just branch-specific nuc and AA mutations? I'm not actually sure if these should be in `infoPanel.click`.

2. It will often be impossible to fit a hover tooltip just given height of Phylogeny panel relative to height of tooltip. But that said, maybe an easy improvement would be to make the hover tooltips slightly wider to fit more content into the space? I'm rarely getting an issue of occlusion on left-side of panel or right-side of panel.

**Comment by emmahodcroft** [comment]
I agree that I think `divergence` is the only one, at least for `ncov` - I didn't check the code to see if other fields (which maybe we don't have in `ncov`) are always in both. Moving this to be in both would be great. I agree that the branch-specific mutations isn't worth adding - this is mostly useful in mouse-over context anyway, and as it's near the top it's less-affected. 

I'd also be happy to make the boxes a bit wider - I agree it's rarely width that's constraining the view in my experience, mostly height.

**Comment by corneliusroemer** [comment]
I've also got this now on Brave macOS 14.0
<img width="1056" alt="image" src="https://github.com/nextstrain/auspice/assets/25161793/73876f63-3456-44b2-8184-a9a549a5604e">

Any branch hover tooltips cut off at the entropy panel.

Auspice 2.48.0
